crates/ruff_cli/resources/test/fixtures/include-test/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.ruff]
+include = ["a.py", "subdirectory/c.py"]
```

Comment:
Nit: do you mind adding a newline here, so it isn't automatically added next time someone edits the file?